multicluster/build/yamls/antrea-multicluster-leader-namespaced.yml

```diff
@@ -45,6 +55,26 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - crd.antrea.io
+  resources:
+  - clusternetworkpolicies
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
does leader need any cnp permission?

Reply:
No it does not. However this rbac manifest is generated and the only place I've added cnp permissions is in resourceimport_controller.go:
`//+kubebuilder:rbac:groups=crd.antrea.io,resources=clusternetworkpolicies,verbs=get;list;watch;create;update;patch;delete`
Not sure how to strip this permission from leader cluster specifically. @luolanzone any ideas?

Reply:
When we split them into member and leader manifests, we didn't separate these accesses setting in fine-grained way, I think we may need a follow up PR to fix these access right. I will create an issue for this first.
@tnqn  do you think it's OK to fix this kind of issue in a new PR?

Reply:
Yes, but please address it in release 1.6. We had such issues before and should ensure all components only get permissions they require.